Package.swift

```diff
@@ -19,7 +19,10 @@ import PackageDescription
 var packageDependencies: [Package.Dependency] = [
   .package(url: "https://github.com/apple/swift-protobuf.git", .upToNextMinor(from: "1.1.1")),
   .package(url: "https://github.com/kylef/Commander.git", from: "0.8.0"),
-  .package(url: "https://github.com/apple/swift-nio-zlib-support.git", from: "1.0.0")
+  .package(url: "https://github.com/apple/swift-nio-zlib-support.git", from: "1.0.0"),
+  .package(url: "https://github.com/apple/swift-nio.git", from: "1.9.0"),
+  .package(url: "https://github.com/apple/swift-nio-nghttp2-support.git", from: "1.0.0"),
+  .package(url: "https://github.com/lukasa/swift-nio-http2.git", .branch("cb-issue-24"))
```

Comment:
We shouldn't be pointing at a fork here

Reply:
Done (the required patch had not yet been submitted).